frontend/src/app/oe/interfaces/oe-energy.interface.ts

```diff
@@ -155,6 +155,20 @@ export interface BlockTimeStrikeGuessPublic {
   guess : 'slow' | 'fast';
 }
 
+export interface BlockTimeStrikeGuessResultPublic {
+  person : string;
+  strike : BlockTimeStrike;
+  creationTime : number;
+  archiveTime : number;
+  guess : 'slow' | 'fast';
+  observedResult : 'slow' | 'fast';
+}
+
+export interface BlockTimeStrikeResult {
```

Comment:
there is a `BlockTimeStrikePublic` with the same members exist already, please use it instead of new one.

Reply:
Done